sdk/src/store.rs

```diff
@@ -2461,6 +2520,7 @@ impl Store {
     /// asset and manifest. Updates XMP with provenance record.
     /// When called, the stream should contain an asset matching format.
     /// Returns a tuple (output asset, manifest store) with a `Vec<u8>` containing the output asset and a `Vec<u8>` containing the insert manifest store.  (output asset, )
+    #[allow(dead_code)]
```

Comment:
Consider adding a note explaining why the `dead_code` bypass is needed and when/if we might be able to remove it.

Reply:
changed to being marked as a v1 feature